LICENSE

```diff
@@ -1,21 +1,6 @@
-MIT License
+Source code in this repository is variously licensed under the Testkube 
+Community License (CCL) and the MIT license. 
 
-Copyright (c) 2022 Kubeshop
-
-Permission is hereby granted, free of charge, to any person obtaining a copy
-of this software and associated documentation files (the "Software"), to deal
-in the Software without restriction, including without limitation the rights
-to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
-furnished to do so, subject to the following conditions:
-
-The above copyright notice and this permission notice shall be included in all
-copies or substantial portions of the Software.
-
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-SOFTWARE.
+A copy of each license can be found in the licenses directory. Source code 
+in a given file is licensed under the MIT and the copyright belongs to 
```

Comment:
Source code in a given file is not necessarily licensed under MIT. 
New language:
"Source code in a given file is licensed under the applicable license for that source code (either TCL or MIT), which can be found in the LICENSE file referenced at the beginning of the file."

Reply:
Potential alternate draft given no header for MIT-licensed source code files

Source code in a given file is licensed under the applicable license for that source code. Source code is licensed under the MIT license unless otherwise indicated in the header referenced at the beginning of the file.

Reply:
pushed what we agreed to on slack